src/Compilers/CSharp/Test/Syntax/Parsing/CollectionExpressionParsingTests.cs

```diff
@@ -18419,4 +18419,343 @@ void M()
             }
             """).VerifyDiagnostics();
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/75318")]
```

Comment:
>WorkItem("https://github.com/dotnet/roslyn/issues/75318") [](http://example.com/codeflow?start=11&length=57)

Is this issue related? If not, let's remove the `[WorkItem]` attribute from the added tests.

Reply:
done.